bibliopixel/animation/feedback.py

```diff
@@ -0,0 +1,47 @@
+import copy
+from . collection import Collection
+from .. util import color_list
+
+
+class Feedback(Collection):
```

Comment:
Arg...keeping cool new animations hostage in the non-released, unstable branch :P Khaaaaan!